kernel/src/arch/x86/smp/smp.cpp

```diff
@@ -27,6 +26,7 @@
 #include <user/elf_load.h>
 #include <user/syscall/sys_io.h>
 #include <utility>
+void poll_keystroke();
```

Comment:
Put this in ps2.h

Reply:
Keep in mind that you need the `poll_keystroke` function in the namespace in both the header and the source so that it links properly